teuthology/orchestra/remote.py

```diff
@@ -679,6 +679,15 @@ def is_vm(self):
             self._is_vm = teuthology.lock.query.is_vm(self.name)
         return self._is_vm
 
+    @property
+    def is_container(self):
+        if not hasattr(self, '_is_container'):
+            self._is_container = not bool(self.run(
+                args="ls /.dockerenv || ls /run/.containerenv",
```

Comment:
ls?

Reply:
Just checking for the existence of either file.

Reply:
maybe `test` is better?

Reply:
Sure, updatedd